db/migrate/20210607111249_add_proposal_form_id_in_proposals.rb

```diff
@@ -0,0 +1,5 @@
+class AddProposalFormIdInProposals < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :proposals, :proposal_form, null: false, foreign_key: true
+  end
+end
```

Comment:
This causes an error for me:

```
proposals | == 20210607111249 AddProposalFormIdInProposals: migrating =====================
proposals | -- add_reference(:proposals, :proposal_form, {:null=>false, :foreign_key=>true})
db_1   | 2021-06-07 17:11:59.013 UTC [36] ERROR:  column "proposal_form_id" of relation "proposals" contains null values
db_1   | 2021-06-07 17:11:59.013 UTC [36] STATEMENT:  ALTER TABLE "proposals" ADD "proposal_form_id" bigint NOT NULL
proposals | rake aborted!
proposals | StandardError: An error has occurred, this and all later migrations canceled:
proposals | 
proposals | PG::NotNullViolation: ERROR:  column "proposal_form_id" of relation "proposals" contains null values
```

The migration will need to account for existing data (which now exists on the production server). Maybe remove the `null: false` clause, do a query to get a valid proposal_form id, then update the new proposal_form_id columns with the reference, then change_column to add the not null requirement?